application/src/main/webapp/vue-app/components/ExoChatContact.vue

```diff
@@ -163,6 +163,7 @@ export default {
       return this.type === 'u' && !this.isEnabled ? 'inactive' : 'active';
     },
     spaceGroupUri() {
+      this.getSpace();
```

Comment:
If this function is not reused somexhere else, just put the instructions here and remove it.